README.md

```diff
@@ -23,6 +23,18 @@ To sync a wallet to the default electrum server:
 cargo run -- wallet --descriptor "wpkh(tpubEBr4i6yk5nf5DAaJpsi9N2pPYBeJ7fZ5Z9rmN4977iYLCGco1VyjB9tvvuvYtfZzjD5A8igzgw3HeWeeKFmanHYqksqZXYXGsw5zjnj7KM9/*)" sync
 ```
 
+To sync a wallet to Bitcoin Core node (assuming a regtest node at 127.0.0.1:18444) serving compact filters:
```

Comment:
Since it takes a very long time to compile `librocksdb` I would add a little warning here in the README. Something like "keep in mind that enabling this feature will make building the projects take a few more minutes, as compact filters require RocksDB which takes a while to build".

Reply:
It did take longer in my machine, but nor considerably. Adding the note. @notmandatory noted. Updated in revised.